frontend/src/features/admin-form/create/workflow/components/WorkflowContent/InactiveStepBlock/InactiveStepBlock.tsx

```diff
@@ -24,12 +27,63 @@ interface InactiveStepBlockProps {
   handleOpenDeleteModal: () => void
 }
 
+interface RespondentBadgeProps {
+  step: FormWorkflowStepDto
+  idToFieldMap: Dictionary<FormFieldWithQuestionNo<FormField>>
+}
+const FirstStepRespondentBadge = ({
+  step,
+  idToFieldMap,
+}: RespondentBadgeProps): JSX.Element | null => {
+  if (step.workflow_type === WorkflowType.Static && step.emails.length > 0) {
+    throw new Error('First step must not have static emails.')
```

Comment:
Can throwing be avoided?

There's no way the user can self-correct and makes the workflow page completely inaccessible. In general, we don't throw on FE except on certain cases such as:
1.  `Provider/Context`: where the page completely fails to be operable
2. `mutations`: where the query has a capture on thrown errors, and a toast or feedback is shown

Reply:
It feels like this is a data assertion, which does not impact whether the contents below could be shown.